interface/src/app/tribe_client/genesets.js

```diff
@@ -0,0 +1,64 @@
+angular.module('adage.tribe_client.genesets', [
+  'adage.tribe_client.resource'
+])
+
+.directive('genesetSearchForm', [function() {
+  return {
+    controller: ['$scope', function($scope) {
+      // Number of results (in this case genesets) to come back
+      // in each response page.
+      $scope.limit = 10;
```

Comment:
Ah good catch. I agree - I think this should probably get set in the genesetSearchForm directive controller, since that is where all the business logic for this directive is handled. I will remove the setting of this ``limit`` parameter wherever this directive gets called.